server/src/webserver.rs

```diff
@@ -645,18 +656,30 @@ async fn serve_file(
         source_lexer(&file_contents, lexer)
     };
 
-    // <p>Convert doc blocks from Markdown to HTML</p>
+	// <p>Convert doc blocks from Markdown to HTML then parse using HTML5Ever</p>
```

Comment:
Avoid using tabs -- only indent with spaces.

Reply:
This was fixed with cargo fmt.  Please see updated code.